be/src/column/stream_chunk.h

```diff
@@ -55,7 +55,25 @@ struct BinlogOffset {
     int64_t lsn;
 };
 
+class TMVMaintenanceTasks;
 class TMVStartEpochTask;
+
+/**
+ *  `MVMaintenanceTaskInfo` contains the basic MV maintenance tasks info for all task types.
+ */
+struct MVMaintenanceTaskInfo {
```

Comment:
Is it necessary to duplicate all these structures with thrift? 
If necessary, should we put these tasks into a separated header, instead of the `stream_chunk.h` ?

Reply:
FE's `onReportEpoch` requires those for now. Maybe we can rethink this later?

```
    public void onReportEpoch(TMVMaintenanceTasks request) {
        LOG.info("onReportEpoch: {}", request);
        Preconditions.checkArgument(request.isSetDb_id(), "required");
        Preconditions.checkArgument(request.isSetMv_id(), "required");
        Preconditions.checkArgument(request.isSetTask_id(), "required");
        Preconditions.checkArgument(request.isSetReport_epoch(), "must be report");

```